ddl/column_test.go

```diff
@@ -121,7 +121,9 @@ func (s *testColumnSuite) TestColumn(c *C) {
 	c.Assert(err, IsNil)
 
 	i := int64(0)
-	t.IterRecords(ctx, t.FirstKey(), t.Cols(), func(h int64, data []interface{}, cols []*column.Col) (bool, error) {
+	txn, err := ctx.GetTxn(false)
```

Comment:
U can use `ctx.GetTxn(true)`, it inner call `FinishTxn `, then u can remove line 120 to 121.